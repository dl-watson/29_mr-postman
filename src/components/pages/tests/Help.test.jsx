import React from "react";
import { render, cleanup } from "@testing-library/react";
import Help from "../Help";

describe("Help component", () => {
  afterEach(() => cleanup());
  it("renders Help", () => {
    const { asFragment } = render(<Help />);
    expect(asFragment()).toMatchSnapshot();
  });
});
