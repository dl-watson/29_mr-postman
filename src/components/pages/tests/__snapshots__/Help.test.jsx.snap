// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help component renders Help 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
